Hestia/Hestia/backend/utils/server_discovery/AutoServerDicovery.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Foundation;
+using System;
+using UIKit;
+
+namespace Hestia.backend.utils.server_discovery
+{
+    /// <summary>

```

Comment:
More of this documentation in you code would be nice, you have some big classes which could benefit from some better documentation.